src/diffusers/models/embeddings.py

```diff
@@ -249,6 +249,34 @@ def forward(self, x):
         return out
 
 
+class PositionalEmbedding(nn.Module):
```

Comment:
Why can't we leverage: https://github.com/huggingface/diffusers/blob/33293ed50470f681ad8c704e325b2960dc66cfce/src/diffusers/models/embeddings.py#L26 here?

Reply:
These are sinoidal position embeddings, let's maybe name them differently